Add Password Generator Script

AreebaTahir00 opened the pull request:
Uploading the initial version of password generator script.

AreebaTahir00 committed:
Add files via upload

Uploading the initial version of password generator script.

SIBTAIN-ASAD reviewed (commented):
Can you add comments and do strings.

AreebaTahir00 committed:
Added comments and docstrings

- Added docstrings for all functions
- Wrote inline comments for better understanding
- Wrapped logic inside main() and used if __name__ == "__main__"

AreebaTahir00 committed:
Update README.md

AreebaTahir00 committed:
Update README.md

AreebaTahir00 committed:
Update README.md

AreebaTahir00 committed:
Update README.md

AreebaTahir00 committed:
Update README.md

SIBTAIN-ASAD reviewed (approved):
LGTM
